### DISCUSSION TITLE
Ajarin Cara Konsumsi APInya di PHP

### DISCUSSION THREAD

**Discussion by aminubilhuda** [kirimwa-id/forum#110] [Umum]
Ada yang bisa ajarin make apinya di php soalnya bingung banget "AWAM BANGET" Iingin Belajar

**Comment by rioastamal** [comment]
Silahkan pelajari dulu tentang REST API dan bagaimana melakukan HTTP request di PHP. 

Kemudian baca referensi API.
https://developer.kirimwa.id/api/v1/

Dan tutorial:
https://github.com/kirimwa-id/forum/discussions/77
